pages/dao/[symbol]/proposal/components/instructions/Raydium/AddLiquidityToPool.tsx

```diff
@@ -0,0 +1,261 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import React, { useContext, useEffect, useState } from 'react'
+import BigNumber from 'bignumber.js'
+import * as yup from 'yup'
+import { BN } from '@project-serum/anchor'
+import {
+  Governance,
+  ProgramAccount,
+  serializeInstructionToBase64,
+} from '@solana/spl-governance'
+import { PublicKey } from '@solana/web3.js'
+import Input from '@components/inputs/Input'
+import Select from '@components/inputs/Select'
+import useGovernedMultiTypeAccounts from '@hooks/useGovernedMultiTypeAccounts'
+import useRealm from '@hooks/useRealm'
+import { createAddLiquidityInstruction } from '@tools/sdk/raydium/createAddLiquidityInstruction'
+import {
+  getAmountOut,
+  getLiquidityPoolKeysByLabel,
+} from '@tools/sdk/raydium/helpers'
+import { liquidityPoolKeysList } from '@tools/sdk/raydium/poolKeys'
+import { debounce } from '@utils/debounce'
+import { isFormValid } from '@utils/formValidation'
+import {
+  AddLiquidityRaydiumForm,
+  UiInstruction,
+} from '@utils/uiTypes/proposalCreationTypes'
+import useWalletStore from 'stores/useWalletStore'
+
+import { NewProposalContext } from '../../../new'
+import GovernedAccountSelect from '../../GovernedAccountSelect'
+
+const AddLiquidityToPool = ({
```

Comment:
We can't really go on like this.

We need:
- better component props API
- some kind of minimal form abstraction
- some kind of minimal instruction form abstraction on top
- better helpers related to adding a new instruction